### PR TITLE
Windowmon

### PR DESCRIPTION
**Pull request by skvl**

### CODE REVIEW

**Comment by skvl** [comment]
@tklengyel is something wrong with this plugin?

**Comment by tklengyel** [comment]
Looks like you included your other new plugin in this PR too

**Comment by skvl** [comment]
Yes. I have done dome work each time based on previous. I will re-base it each time or we could combine them into single. As you wish.

**Comment by tklengyel** [comment]
I'll just merge this after clipboardmon is merged.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Comment by tklengyel** [comment]
@skvl This will need to be rebased to resolve the conflicts

**Comment by skvl** [comment]
Done!

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/plugins/windowmon/windowmon.cpp [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    {
+        PRINT_DEBUG("Rekall profile: no '%s' found\n", constant);
+        return false;
+    }
+
+    *rva = json_object_get_int64(jsymbol);
+    return true;
+}
+
+struct SYSTEM_SERVICE_TABLE_32
```
Comment (tklengyel) [inline src/plugins/windowmon/windowmon.cpp]: Please move these structure definitions into a `private.h`

Comment (tklengyel) [reply src/plugins/windowmon/windowmon.cpp]: In fact, I don't see these structures actually being used for anything other then to calculate their size, so you might as well just define their size in a `#define`

Comment (skvl) [reply src/plugins/windowmon/windowmon.cpp]: Done.

**Pull request merged**